### PR TITLE
Added Countdown for DMA on home page

### PR DESCRIPTION
**Pull request by gericci**
## Summary of Changes

1. [x] Fixed missing quotes on home page
2. [x] Create partial include "dma-countdown.njk"
1. [x] Added include to aside.njk
1. [x] Updated stylesheets with new DMA block
-->

### CODE REVIEW

**Comment by gericci** [comment]
In order to put the countdown above the fold in small screens, I've duplicated the block (not possible to do otherwise in a robust manner). The problem is that the Javascript is called twice (not sure why). If somebody can take a look.

**Comment by rejhgadellaa** [comment]
Hmm ok the current implementation has two main issues:

1) The `dma-countdown.njk` is included twice, which means the `<script />` in there runs twice. The same script is declared again in `home.html`.

2) The html/script uses `getElementById`, but that will always return a single element (I think the last one in the DOM). So that needs a rewrite.

I'll push a hotfix that addresses these issues but it's still far from perfect. If anyone has more time to improve it, that'd be great :) If not, we at least have a working version :)

File: src/_includes/partials/dma-countdown.njk [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      </span>m:<span id="remaining-secs">
+        00
+      </span>s
+    </p>
+  </div>
+  <p><a href="">Read more about the Digital Markets Act</a></p>
+</div>
+
+
+<script>
```
Comment (rejhgadellaa) [inline src/_includes/partials/dma-countdown.njk:22]: Change #1: this `<script />` needs to go, only keep the one in `home.html`

File: src/pages/home.html [hunk 2]
```diff
@@ -140,3 +143,57 @@ <h2>How can you help?</h2>
 ## Blog Posts
 Here are the most recent posts we have published on our blog:
 -->
+
+
+<script>
```
Comment (rejhgadellaa) [inline src/pages/home.html:148]: Change # 2: Replace the `<script />` tag here with:

```javascript
<script>
  (() => {

    [...document.querySelectorAll('.dma-countdown')].forEach(
      parent => {

        function countDown(endtime) {
          const total = Date.parse(endtime) - Date.parse(new Date());
          const seconds = Math.floor((total / 1000) % 60);
          const minutes = Math.floor( (total/1000/60) % 60 );
          const hours = Math.floor( (total/(1000*60*60)) % 24 );
          const days = Math.floor( total/(1000*60*60*24) );

          cdDays.innerText = days;
          cdHours.innerText = ('0' + hours).slice(-2);
          cdMins.innerText = ('0' + minutes).slice(-2);
          cdSecs.innerText = ('0' + seconds).slice(-2);

          return {
            days,
            hours,
            minutes,
            seconds
          };
        }

        function initializeClock(endtime) {
          countDown(endtime);
          const timeinterval = setInterval(() => {
            const t = countDown(endtime);

            if (t.days <= 0) {
              dayCd.style.display = "none";
            }

            if (t.total <= 0) {
              clearInterval(timeinterval);
            }
          }, 1000);
        }

        const noscript = parent.querySelector('#deadline');
        const deadline = '2024-03-06T01:00:00Z';
        const timer = parent.querySelector('div.timer');
        const dayCd = parent.querySelector('#dayCd');
        const timeCd = parent.querySelector('#timeCd');
        const cdDays = parent.querySelector('#remaining-days');
        const cdHours = parent.querySelector('#remaining-hours');
        const cdMins = parent.querySelector('#remaining-mins');
        const cdSecs = parent.querySelector('#remaining-secs');

        // If Js on
        noscript.style.display = 'none';
        timer.style.display = 'flex';

        initializeClock(deadline);

      }
    )

  })();
</script>
````

**Comment by rejhgadellaa** [comment]
Close in favor of #88

**Pull request closed without merging**